app/_redirects

```diff
@@ -125,28 +125,46 @@
 
 
 # KONNECT
-/konnect/org-management/okta-role-mapping           /konnect/org-management/okta-idp/#map-roles-to-groups
-/konnect/access-beta                                /konnect/access-account
+
 /konnect/service-hub                                /konnect/servicehub/
 /konnect/service-hub/*                              /konnect/servicehub/:splat
-/konnect/runtime-manager/kong-gateway-runtime       /konnect/runtime-manager/#kong-gateway
-/konnect/servicehub/plugins/enable-service-plugin/  /konnect/manage-plugins/enable-service-plugin/
-/konnect/servicehub/plugins/enable-route-plugin/    /konnect/manage-plugins/enable-route-plugin/
-/konnect/dev-portal/administrators/customization/themes         /konnect/dev-portal
+/konnect/org-management/okta-role-mapping           /konnect/org-management/okta-idp/#map-roles-to-groups
+/konnect/key-concepts-and-terms/                    /konnect/
+/konnect/reference/konnect-api/                     /konnect/
+/konnect/reference/proxy-traffic/                   /konnect/
```

Comment:
Looks like this one is still redirecting to /konnect/reference/proxy-traffic/ instead of /konnect/

Reply:
Huh. That's weird. Let me just delete the topic and see if that makes a difference.